Index: breeze-plymouth-5.6.3/breeze/breeze.script
===================================================================
--- breeze-plymouth-5.6.3.orig/breeze/breeze.script
+++ breeze-plymouth-5.6.3/breeze/breeze.script
@@ -32,18 +32,18 @@
 
 // -------------------------------- Assets ---------------------------------- //
 
-global.title.text = "Plasma 5.6";
+global.title.text = "KDE neon Developer Edition (unstable branches)";
 global.defaults.font.default = "Noto Sans 12";
 global.defaults.font.title = "Noto Sans 14";
 
 global.assets = [];
 if (Window.GetBitsPerPixel() == 4) {
-    assets.logo          = "images/16bit/plasma.logo.png";
+    assets.logo          = "images/16bit/neon.logo.png";
     assets.text_input    = "images/16bit/text-input.png";
 
     assets.spinner_base  = "images/16bit/spinner";
 } else {
-    assets.logo          = "images/plasma.logo.png";
+    assets.logo          = "images/neon.logo.png";
     assets.text_input    = "images/text-input.png";
 
     assets.spinner_base  = "images/spinner";
