Index: breeze-plymouth/breeze/breeze.script
===================================================================
--- breeze-plymouth.orig/breeze/breeze.script
+++ breeze-plymouth/breeze/breeze.script
@@ -38,12 +38,12 @@ global.defaults.font.title = "Noto Sans
 
 global.assets = [];
 if (Window.GetBitsPerPixel() == 4) {
-    assets.logo          = "images/16bit/plasma.logo.png";
+    assets.logo          = "images/16bit/neon.logo.png";
     assets.text_input    = "images/16bit/text-input.png";
 
     assets.spinner_base  = "images/16bit/spinner";
 } else {
-    assets.logo          = "images/plasma.logo.png";
+    assets.logo          = "images/neon.logo.png";
     assets.text_input    = "images/text-input.png";
 
     assets.spinner_base  = "images/spinner";
@@ -72,8 +72,8 @@ colors.neon_green = Color(0.1020, 0.7373
 colors.neon_blue = Color(0.1608, 0.5020, 0.7255);
 
 global.palette = [];
-palette.background.top = colors.plasma_blue;
-palette.background.bottom = colors.icon_blue;
+palette.background.top = colors.neon_green;
+palette.background.bottom = colors.neon_blue;
 palette.text.normal = colors.paper_white;
 palette.text.tinted = colors.paper_white;
 palette.text.action = colors.paper_white;
Index: breeze-plymouth/breeze-text/breeze-text.plymouth.cmake
===================================================================
--- breeze-plymouth.orig/breeze-text/breeze-text.plymouth.cmake
+++ breeze-plymouth/breeze-text/breeze-text.plymouth.cmake
@@ -4,4 +4,4 @@ Description=A Breeze splash for text mod
 ModuleName=breeze-text
 
 [breeze-text]
-title=Plasma @PROJECT_VERSION@
+title=KDE neon @PROJECT_VERSION@
